code/modules/events/aurora_caelus.dm

```diff
@@ -0,0 +1,32 @@
+/datum/event/aurora_caelus
+	has_skybox_image = TRUE
+	announceWhen = 1
+	startWhen = 5
+	endWhen = 55
+
+/datum/event/aurora_caelus/announce()
+	command_announcement.Announce("[station_name()]: A harmless cloud of ions is approaching your station, and will exhaust their energy battering the hull.\
+	Nanotrasen has approved a short break for all employees to relax and observe this very rare event.\
+	During this time, starlight will be bright but gentle, shifting between quiet green and blue colors.\
+	Any staff who would like to view these lights for themselves may proceed to the area nearest to them with viewing ports to open space.\
+	We hope you enjoy the lights.", "Nanotrasen Meterology Divison", new_sound = 'sound/misc/notice2.ogg')
+	for(var/V in player_list)
+		var/mob/M = V
+		if(affecting_z)

```

Comment:
How does that check in any way affect anything again? It will always be true.

Reply:
Am currently testing alternatives - I also realized that the "announce" would play the sound for all mobs instead of when it actually starts, so I'm going to fix that too.